src/embed/sphere-renderer.js

```diff
@@ -148,7 +157,30 @@ SphereRenderer.prototype.createPhotosphere_ = function(texture, opt_params) {
     }
   }
 
-  var material = new THREE.MeshBasicMaterial({ map: texture });
+  var material;
+  if (texture.format === THREE.RGBAFormat && texture.flipY === false) {
```

Comment:
Yeah. This is a temporary solution for [the webkit bug](https://bugs.webkit.org/show_bug.cgi?id=164540). Indeed it should be fixed in webkit.